### PR TITLE
Fixed extra slash in url while navigating in certain conditions

### LINKED ISSUE #64: Link double slash
In some palces in the UI if going back via browser navigation then clicking links we sometimes see double slashes in the URL breaking the app. 
Maybe we can just check for double slashes and remove, think the issue is from hash router maybe. 

Will try to add more details if seen again

### PR DESCRIPTION
**Pull request by dcrescimbeni**
# Description

Closes #64 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Manual testing. All previous tests passed.

# Checklist:

- [x] My code follows the existing style of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any UI changes have been tested and made responsive for mobile views

### CODE REVIEW

File: src/Components/GuildCard/__snapshots__/GuildCard.test.tsx.snap [hunk 1]
```diff
@@ -188,7 +188,7 @@ exports[`GuildCard Should render with full parameters 1`] = `
   <a
     class="sc-bdfBQB hcWJCy"
     data-testid="guildCard"
-    href="//0xa47BbE8Dd6dB29D45FE5eeD838c4f136884AEAF3"
+    href="/0xa47BbE8Dd6dB29D45FE5eeD838c4f136884AEAF3"
```
Comment (rossneilson) [inline src/Components/GuildCard/__snapshots__/GuildCard.test.tsx.snap:191]: Wow, a test was picking this up!?

**Comment by rossneilson** [comment]
There are still a few places we are not using the new util, should we add this everywhere?

**Comment by dcrescimbeni** [comment]
> There are still a few places we are not using the new util, should we add this everywhere?

The bug was in the logic of the forward navigation: it took the current URL (`location.pathname`) and added a "/newPage" at the end of the URL. If the URL already ended in a slash, then it ended with a double slash and an error.

This isn't the case for backward navigation, since it navigates to a fixed URL, like `/chain/guildUrl`, and doesn't use the current location.

So I don't think we should add it to all navigation logic.

**Comment by rossneilson** [comment]
> > There are still a few places we are not using the new util, should we add this everywhere?
> 
> The bug was in the logic of the forward navigation: it took the current URL (`location.pathname`) and added a "/newPage" at the end of the URL. If the URL already ended in a slash, then it ended with a double slash and an error.
> 
> This isn't the case for backward navigation, since it navigates to a fixed URL, like `/chain/guildUrl`, and doesn't use the current location.
> 
> So I don't think we should add it to all navigation logic.

Ok, I was thinking more just for consistency and being able to easily change behaviour everywhere in future but if its not needed we can leave it

**Review by rossneilson — APPROVED** [review]

**Pull request merged**